lib/seahorn/BvOpSem2MemManagerMixin.hh

```diff
@@ -306,25 +309,103 @@ public:
   }
 
   Expr getFatData(PtrTy p, unsigned SlotIdx) override {
-    auto res = base().getFatData(BasePtrTy(std::move(p)), SlotIdx);
-    return res;
+    return hana::eval_if(
+        MemoryFeatures::has_fatmem(hana::type<BaseT>{}),
+        [&](auto _) {
+          auto res = _(base()).getFatData(BasePtrTy(std::move(p)), SlotIdx);
+          return res;
+        },
+        [&] {
```

Comment:
no need for brackets?

Reply:
nothing being passed in since we just print an error message.